apps/studio/components/interfaces/Organization/BillingSettings/Subscription/ExitSurveyModal.tsx

```diff
@@ -45,10 +45,6 @@ const ExitSurveyModal = ({ visible, subscription, projects, onClose }: ExitSurve
   const projectsWithComputeDowngrade = projects.filter((project) => {
     const computeSizesThatDoNotResultInComputeDowngrade = ['nano']
 
-    if (subscription?.nano_enabled === false) {
-      computeSizesThatDoNotResultInComputeDowngrade.push('micro')
-    }
-
```

Comment:
Can be simplified to `projects.filter(({ infra_compute_size }) => infra_compute_size !== 'nano')`?